connPFM/deconvolution/stability_lars_caller.py

```diff
@@ -101,17 +101,18 @@ def run_stability_lars(data, hrf, temp, jobs, username, niter, maxiterfactor):
                     job_idx,
                 )
             )
+            error_output = os.path.join(temp, f"error{job_idx}")
```

Comment:
Since you're using the same file for both error and output, wouldn't it be better to just call it `connPFM_{job_idx}` or something like that?